.github/workflows/stage-build-deploy.yml

```diff
@@ -0,0 +1,19 @@
+name: Build and Deploy - Staging
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
What is the purpose of this deployment if it has the same trigger as the main-build-deploy.yml?

Reply:
That's the existing functionality (file was renamed so shows as an add in the diff)
It'll be removed in https://github.com/SSWConsulting/SSW.Rules.GPT/issues/113

Reply:
Do you think it should be removed then? Is it still creating a duplicate deployment?

Reply:
No, it deploys to both staging and prod at the same time but they are seperate instances in azure

Reply:
If anything it should come out of the prod one, not this on. As @jackreimers mentioned though there's another issue tracking this. The change here is just to stabilise these deployments and separate the functionality, which will facilitate this other change too.